README.md

```diff
@@ -100,6 +100,31 @@ data:
 The `project` parameter is the name of the AWX project that contains the job
 templates that will be used to run the playbooks.
 
+### Throttling configuration
+
+The `throttling` section of the configuration describes how to throttle the
+execution of healing actions. This is intended to prevent _healing storms_ that
+could happen if the same alerts are send repeatedly to the service.
+
+The `interval` parameter controls the time that the service will remember an
+executed healing action. If an action is triggered more than once in the given
+interval it will be executed only the first time. The rest of the times it will
+be logged and ignored. (see `autoheal.yml` for an example.)
+
+The default interval value is one hour. Leaving the `interval` parameter 0 followed by a time unit suffix (i.e "0h" or "0s") will *disable* throttling altogether.
```

Comment:
0s == 0h == 0
better not to use units with 0 here or anywhere else

Reply:
Thanks :+1: